azext_edge/edge/params.py

```diff
@@ -47,6 +55,24 @@ def load_iotops_arguments(self, _):
             "If not defined, the fallback value `azure-iot-operations` will be used. ",
             validator=validate_namespace,
         )
+        context.argument(
+            "confirm_yes",
+            options_list=["--yes", "-y"],
+            arg_type=get_three_state_flag(),
+            help="Confirm [y]es without a prompt. Useful for CI and automation scenarios.",
```

Comment:
why not use the azure cli built in confirm? It should be something we can pass through to the delete function being called and add in a help prompt if we want to add flavor

Reply:
Because we already take a dependency on Rich which supports this functionality. The azure cli built in confirm is part of the command/knack configuration meaning we have less control over it and how we can present in our flows.

If we didn't have a dependency on Rich then the Knack version could be considered.